idaes/models/properties/modular_properties/base/generic_property.py

```diff
@@ -2424,9 +2097,13 @@ def build(self):
         ):
 
             t_units = self.params.get_metadata().default_units.TEMPERATURE
+            if self.temperature.value is not None:
+                t_value = value(self.temperature)
+            else:
+                t_value = None
```

Comment:
If this is None, does the model break or is there a catch for this case?

Reply:
This just affects whether we can get initial guesses for Teq. Whether the model will solve or not is a separate issue at this point; the goal here is for the code to work without encountering a terminal error.